src/librustc_back/target/emscripten_base.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[cfg(not(windows))]
+pub fn cmd(name: &str) -> String {
+    name.to_string()
+}
+
+#[cfg(windows)]
```

Comment:
Instead of `#[cfg]`, could this use `cfg!` perhaps to get type checking everywhere? Something like:

```rust
pub fn cmd(name: &str) -> String {
    if cfg!(windows) {
        format!("{}.bat", name)
    } else {
        name.to_string()
    }
}
```